src/Toolkit/Toolkit.UI.Controls/FloorFilter/FloorFilter.cs

```diff
@@ -341,7 +341,7 @@ private void HandleControllerPropertyChanges(object? sender, PropertyChangedEven
                     _autoVisibilityWrapper?.SetValue(VisibilityProperty, _controller.ShouldDisplayFloorPicker ? Visibility.Visible : Visibility.Collapsed);
                     break;
                 case nameof(_controller.AllDisplayLevelsSelected):
-                    OnPropertyChanged(nameof(AllDisplayLevelsSelecteded));
```

Comment:
The old property is still changing too.

Reply:
Do you mean we want the old property to keep changing? So undo the deletion of line 344? I'm not sure I follow.

Reply:
For the sake of backwards compatibility, for all the software that has worked around our old misspelling, we should keep firing a PCE for that old property name too &mdash; in addition to the new correctly-spelled PCE.